src/amo/containers/App.js

```diff
@@ -102,13 +86,8 @@ export class AppBase extends React.Component {
   }
 }
 
-export const setupMapStateToProps = (_window) => (state) => ({
-  clientApp: state.api.application,
```

Comment:
This had a bug in it (looking for `application` instead of `clientApp`) so there was never an `h1` on the homepage :(